src/assertions/text.spec.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env mocha */
+/* eslint-disable padded-blocks, no-unused-expressions, max-len, no-console */
+
+import './text';
+import { shallow, mount } from 'enzyme';
+import React from 'react';
+
+const TextFixture = () => (
+  <div className="special burger">Content here</div>
+);
+
+describe('Should enzyme add check text', () => {
+  const shouldEnzyme = should;
+  it('should have new method text', () => {
+    shouldEnzyme.should.have.property('text');
+  });
+  it('should have text TextFixture', () => {
+    [shallow, mount].forEach(method => {
```

Comment:
I prefer to put the `it` inside of the `forEach` with a label of which `render method` it's using. This will make it clear which one is failing, making it easy to debug.